pkg/scheduler/reconciliation/reconciliation.go

```diff
@@ -90,7 +90,7 @@ func prios(opsByPrio map[int64][]*model.OperationEntity, reverse bool) []int64 {
 	return prios
 }
 
-//opGroupType finds out the operation type on a group of operations with the same scheduling ID.
+//opGroupType finds out the operation type on a group of operations with the same scheduling WorkerPoolID.
```

Comment:
Please explain why it is nor because of the `workerpoolid`?
```suggestion
//opGroupType finds out the operation type on a group of operations with the same schedulingID or  WorkerPoolID.
```

Or is it for both?

Reply:
Oh yes. Sorry about this confusing change. This is a typo. There should be no change here. I fixed it!